install-deps.sh

```diff
@@ -253,6 +258,67 @@ EOF
     fi
 }
 
+function populate_wheelhouse() {
```

Comment:
Right. FWIW, I have ccache enabled in the container workflow, based on your original patches, but unfortunately it doesn't seem to work as desired. It's one of those things I want to look into soon.  But since I don't want to keep sitting on all the work done so far this PRs about getting some of the script improvements out sooner. :-)